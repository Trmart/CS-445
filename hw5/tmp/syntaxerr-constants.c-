## C-S23
int 45234, -34;
char '', 'dogs';
int true, false, true;
int "dogs", "cats";
