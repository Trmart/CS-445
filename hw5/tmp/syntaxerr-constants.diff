3,4c3
< ERROR(3): Empty character ''.  Characters ignored.
< ERROR(3): Syntax error, unexpected ',', expecting identifier.
---
> ERROR(3): Syntax error, unexpected character constant '', expecting identifier.
8c7
< Number of errors: 5
---
> Number of errors: 4
