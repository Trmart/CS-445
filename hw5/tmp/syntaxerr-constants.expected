====================================
ERROR(2): Syntax error, unexpected numeric constant "45234", expecting identifier.
ERROR(3): Empty character ''.  Characters ignored.
ERROR(3): Syntax error, unexpected ',', expecting identifier.
ERROR(4): Syntax error, unexpected Boolean constant "true", expecting identifier.
ERROR(5): Syntax error, unexpected string constant "dogs", expecting identifier.
FILE: syntaxerr-constants.c-
Number of errors: 5
Number of warnings: 1
WARNING(3): character is 4 characters long and not a single character: ''dogs''.  The first char will be used.
